mappings/net/minecraft/block/entity/ShulkerBoxBlockEntity.mapping

```diff
@@ -1,10 +1,17 @@
 CLASS net/minecraft/unmapped/C_gauehqwp net/minecraft/block/entity/ShulkerBoxBlockEntity
 	FIELD f_ainnhabf animationProgress F
 	FIELD f_axjnbhcn cachedColor Lnet/minecraft/unmapped/C_arllgqae;
+	FIELD f_ckaomydn CONTAINER_SIZE I
 	FIELD f_fmgzbbgv prevAnimationProgress F
+	FIELD f_gczipwyl ROWS I
 	FIELD f_gvwgoafu animationStage Lnet/minecraft/unmapped/C_gauehqwp$C_tdtslcqu;
+	FIELD f_kesedzww MAX_LID_ROTATION F
+	FIELD f_laidlwhf OPENING_TICK_LENGTH I
```

Comment:
This sounds like it refers to the duration of a single tick, I assume it refers to a duration in (multiple) ticks, perhaps
```suggestion
	FIELD f_laidlwhf OPENING_TICKS I
```
or
```suggestion
	FIELD f_laidlwhf TICKS_TO_OPEN I
```
?